dataprofiler/profilers/profile_builder.py

```diff
@@ -2290,7 +2295,12 @@ def _update_null_replication_metrics(self, clean_samples: Dict) -> None:
             sum_not_null = np.delete(total_row_sum, col_id) - sum_null
 
             mean_null = sum_null / null_count
-            mean_not_null = sum_not_null / true_count
+
+            if true_count == 0:
+                sum_not_null = np.full(len(data.columns) - 1, np.nan)
+                mean_not_null = np.full(len(data.columns) - 1, np.nan)
+            else:
+                mean_not_null = sum_not_null / true_count
```

Comment:
Why not? 
```
sum_not_null = np.full(len(data.columns) - 1, np.nan)
mean_not_null = np.full(len(data.columns) - 1, np.nan)

 if true_count:
    mean_not_null = sum_not_null / true_count
```

Reply:
`sum_not_null` needs to be in an if because its value is updated before and not after

Reply:
so could we do 
```
mean_not_null = np.full(len(data.columns) - 1, np.nan)

 if true_count:
    sum_not_null = np.full(len(data.columns) - 1, np.nan)
    mean_not_null = sum_not_null / true_count
```

Reply:
```
mean_not_null = np.full(len(data.columns) - 1, np.nan)
if not true_count:
    sum_not_null = np.full(len(data.columns) - 1, np.nan)
else:
    mean_not_null = sum_not_null / true_count
```